sys/include/color.h

```diff
@@ -28,13 +29,17 @@ extern "C" {
 #endif
 
 /**
- * @brief Data-structure describing a RGB color
+ * @brief Data-structure describing a ARGB color
  */
-typedef struct {
-    uint8_t r;          /**< red value      [0 - 255] */
-    uint8_t g;          /**< green value    [0 - 255] */
-    uint8_t b;          /**< blue value     [0 - 255] */
-} color_rgb_t;
+typedef union {
+    uint32_t raw;
+    struct __attribute__((packed)) {
```

Comment:
@kaspar030 do you think this attribute is superfluous here? I was not sure whether a 64bit platform e.g. might put padding into the argb struct (non-commit comment version)